BLHeliBootLoad.inc

```diff
@@ -1,8 +1,11 @@
 ; BLHeli bootloader for SiLabs MCUs. Based upon AVRootloader (copyright HR)
 
 XTAL					EQU	25000000
-
-BOOT_START			EQU	1C00h	; Bootloader segment address
+IF MCU_48MHZ == 2
```

Comment:
Should we rename this Define to something else?
MCU_TYPE or something.

Reply:
Good point, thank you. Added it to the [cleanup issue](https://github.com/bird-sanctuary/bluejay/issues/13).
I want to only have minimum changes in this PR, otherwise rebasing all the currently open PRs will be pain.